tests/system.rs

```diff
@@ -1285,29 +1285,37 @@ fn keeps_newlines_in_external_mod_doc() {
     assert_eq!("The mods multiline documentation\n\nwith an empty line", got.docs);
 }
 
+/// Addresses https://github.com/racer-rust/racer/issues/618
 #[test]
-fn issue_618() {
+fn always_get_all_doc_lines() {
```

Comment:
For me, it's more about knowing it's associated with 618 when it fails just by looking at cargo test output.